tests/lib/common.py

```diff
@@ -80,6 +81,8 @@ def execute(command, capture=False, check=False, disable_logger=False):
     into memory, stdout and stderr are only available on the exception if
     `capture` was True.
 
+    `env` is a dictionary of environment vars passed into Popen.
```

Comment:
Would be cool to have proper parameter documentation that sphinx can handle. but can be fixed later.